renderlib/ViewerWindow.cpp

```diff
@@ -20,16 +20,17 @@ ViewerWindow::ViewerWindow(RenderSettings* rs)
   // TODO have a notion of a scene's selection set,
   // and activate tools via the UI to operate
   // on the selection set.
-  // TEST create a tool and activate it
-  // m_activeTool should not be in m_tools
-  // m_activeTool = new MoveTool();
-  // m_activeTool = new RotateTool();
-  // m_tools.push_back(new AreaLightTool());
+  m_areaLightTool = new AreaLightTool();
```

Comment:
This code is showing two different patterns: one where I abstractly add a "disableable" tool to the list of tools (new ScaleBarTool below), and one where I explicitly create a tool and expect outside code to add/remove it as needed.  I'm not sure if future work in this code will unify the two approaches, but both work for now as long as the bookkeeping is done properly.